package.json

```
@@ -52,7 +52,7 @@
         "typescript": "^4.7.4"
     },
     "peerDependencies": {
-        "svelte": "^3.0"
+        "svelte": "^4.0"
```

```suggestion
        "svelte": "^3.0 || ^4.0"
```
To allow the lib to be compatible with Svelte 3.x